translations/i18nCheck.R

```diff
@@ -30,6 +30,9 @@ rEmptyCalls <- subset(rPotData,
 templateMsgError <- subset(rPotData, grepl(pattern = "%\\d\\$", msgid)                                 # match all placeholders like %1$s
                             & (!grepl(pattern = "%1\\$", msgid) | grepl(pattern = "%[a-zA-Z]", msgid)), # match missing %1$ or %s is present
                            select = c("file", "call", "line_number"))
+rErrorCalls <- subset(rPotData,
+                     grepl(pattern = "^gettext\\(.*%.*\\)", call),
```

Comment:
Why do you use start position '^' in the regular expression? The other expressions are without this start position character (cf 'gettext(|f)\\(['\"]['\"]\\)'. Do the strings given by potools::get_message_data always start with gettext (or gettextf)?

Reply:
because this will match also a nested gettext() inside gettextf() but that is another situation to handle.  the `call` in `rPotData`  not always start with gettext but some `stop` `message` call we excluded from translation.

we shouldn't change `gettext(|f)\(['"]['"]\)` because this match empty strings calls.